config/jobs/periodic/docker/periodic-ci-docker.yaml

```diff
@@ -0,0 +1,108 @@
+periodics:
+    - name: periodic-config-docker
+      cluster: k8s-ppc64le-cluster
+      labels:
+        preset-build-docker: "true"
+      cron: 0 0 * * *
+      decorate: true
+      extra_refs:
+        - base_ref: main
+          org: ppc64le-cloud
+          repo: docker-ce-build
+      reporter_config:
+        slack:
```

Comment:
I came across https://github.com/kubernetes-sigs/prow/blob/main/site/content/en/docs/components/core/crier.md#slack-reporter

Seems like we can move all the slack `reporter_config:` section in jobs to https://github.com/ppc64le-cloud/test-infra/blob/master/config/prow/config.yaml#L5 mentioning the `org/repo:` (ppc64le-clou/docker-ce-build:) like in below example:
![image](https://github.com/user-attachments/assets/314c07b6-aee2-4e93-9b33-e171ab6e51bb)